### ISSUE TITLE
Playing scenes from spankbang won't work and freezes my kodi?...

### ISSUE THREAD

**Issue by frankee123** [dobbelina/repository.dobbelina#783]
Sorry in advance if this isn't right, but I seem to have a problem and I want to know if it's just me or not please. I just got a brand new android 11 tv box and everything seems to be working fine except for Spankbang. Whenever I try to play any scene, the circle keeps spinning around and the only thing I can do is quit kodi and open it up again. Can anybody tell me if they are having this same issue please?

**Comment by dobbelina** [comment]
No issues on Windows to play scenes.
You will need to create a logfile as per the instructions in creating a bug report.

**Comment by DJMix-I-Can** [comment]
Also working on my Nvidia Shield.

**Comment by frankee123** [comment]
https://paste.kodi.tv/fotivacite
Ty for the help guys. I have no idea if this is what you are looking for or if I did it right. I tried following the directions but not sure if it will work since I never get any error that pops up. It's anytime I click on a scene from spankbang or it even happened on some other programs trying to play vids. It acts like it's trying to play the vid or scene, but won't load and the only thing I can do is manually shut kodi down then restart it. Any tips would greatly be appreciated, I have never had this issue and already am upset since I am using a brand new android 11 box as well. Ty
